app/views/events/_form.html.erb

```diff
@@ -29,95 +40,90 @@
           <%= f.radio_button 'online', 'true', class: 'location', checked: @event.online?, autocomplete: 'off' %>
           <i class="fa fa-desktop" aria-hidden="true" id="online-symbol"></i> Online
         </label>
+        <label class="btn btn-default <%= 'active' unless @event.online? %>">
+          <%= f.radio_button 'online', 'false', class: 'location', checked: !@event.online?, autocomplete: 'off' %>
```

Comment:
maybe it's best for development speed to remove the `hybrid` option (since we'd need to alter the datamodel).